### ISSUE TITLE
[master] Error when export translation

### ISSUE THREAD

**Issue by maitaoriana** [odoo/odoo#17834]
Impacted versions:
master

Steps to reproduce:
1. Export Translation
2. Select Lenguage
3. File Format = PO file
4. Select App
5. Export 

Current behavior:

```
Traceback (most recent call last):
  File "/home/odoo/runbot/extra/runbot/static/build/240405-master-252d65/odoo/http.py", line 642, in _handle_exception return super(JsonRequest, self)._handle_exception(exception)
  File "/home/odoo/runbot/extra/runbot/static/build/240405-master-252d65/odoo/http.py", line 679, in dispatch result = self._call_function(**self.params)
  File "/home/odoo/runbot/extra/runbot/static/build/240405-master-252d65/odoo/http.py", line 335, in _call_function return checked_call(self.db, *args, **kwargs)
  File "/home/odoo/runbot/extra/runbot/static/build/240405-master-252d65/odoo/service/model.py", line 102, in wrapper return f(dbname, *args, **kwargs)
  File "/home/odoo/runbot/extra/runbot/static/build/240405-master-252d65/odoo/http.py", line 328, in checked_call result = self.endpoint(*a, **kw)
  File "/home/odoo/runbot/extra/runbot/static/build/240405-master-252d65/odoo/http.py", line 923, in __call__ return self.method(*args, **kw)
  File "/home/odoo/runbot/extra/runbot/static/build/240405-master-252d65/odoo/http.py", line 508, in response_wrap response = f(*args, **kw)
  File "/home/odoo/runbot/extra/runbot/static/build/240405-master-252d65/odoo/addons/web/controllers/main.py", line 914, in call_button action = self._call_kw(model, method, args, {})
  File "/home/odoo/runbot/extra/runbot/static/build/240405-master-252d65/odoo/addons/web/controllers/main.py", line 902, in _call_kw return call_kw(request.env[model], method, args, kwargs)
  File "/home/odoo/runbot/extra/runbot/static/build/240405-master-252d65/odoo/api.py", line 681, in call_kw return call_kw_multi(method, model, args, kwargs)
  File "/home/odoo/runbot/extra/runbot/static/build/240405-master-252d65/odoo/api.py", line 672, in call_kw_multi result = method(recs, *args, **kwargs)
  File "/home/odoo/runbot/extra/runbot/static/build/240405-master-252d65/odoo/addons/base/module/wizard/base_export_language.py", line 38, in act_getfile tools.trans_export(lang, mods, buf, this.format, self._cr)
  File "/home/odoo/runbot/extra/runbot/static/build/240405-master-252d65/odoo/tools/translate.py", line 710, in trans_export_process(format, modules, translations, buffer, lang)
  File "/home/odoo/runbot/extra/runbot/static/build/240405-master-252d65/odoo/tools/translate.py", line 684, in _process writer.write(row['modules'], row['tnrs'], src, row['translation'], row['comments'])
  File "/home/odoo/runbot/extra/runbot/static/build/240405-master-252d65/odoo/tools/translate.py", line 625, in write self.buffer.write("#. module%s: %s\n" % (plurial, ', '.join(modules)))
TypeError: 'unicode' does not have the buffer interface
```

Expected behavior:
Export PO file.

![dashboard odoo](https://user-images.githubusercontent.com/17258001/27541945-c8d7b338-5a53-11e7-9a95-199a918179e6.png)

Video/Screenshot link (optional):
https://www.youtube.com/watch?v=djJur9VxV2o&feature=youtu.be

**Comment by luistorresm** [comment]
CC @qdp-odoo @mart-e 

Regards

**Comment by xmo-odoo** [comment]
Should be fixed by 98602e9
